tests/eas/load_tracking.py

```diff
@@ -1273,3 +1290,224 @@ def test_util_tg4_group(self):
         """
         return self._test_group_util('/tg1/tg2/tg3/tg4')
 
+
+class CgroupsMigrationTest(_PELTTaskGroupsTest):
+    """
+    Test PELT utilization for task groups migration.
+    Initial group hierarchy:
+                           +-----+
+                           | "/" |
+                           +-----+
+                           /     \
+                          /       \
+                      +------+   +------+
+                      |"/tg1"|   |"/tg2"|
+                      +------+   +------+
+                       /    \         \
+                      /      \         \
+                    t1_1    t1_2      t2_1
+
+    Final group hierarchy:
+                           +-----+
+                           | "/" |
+                           +-----+
+                           /     \
+                          /       \
+                      +------+   +------+
+                      |"/tg1"|   |"/tg2"|
+                      +------+   +------+
+                       /          /    \
+                      /          /      \
+                    t1_1       t1_2    t2_1
+    """
+    target_cpu = 0
+    root_group = None
+    trace = None
+
+    @classmethod
+    def _getExperimentsConf(cls, test_env):
+        # Run all workloads on a CPU with highest capacity
+        cls.target_cpu = [min(test_env.calibration(),
+                              key=test_env.calibration().get)]
+        period_ms = 16
```

Comment:
Same comment as below (sorry, I've started the review from the bottom)... cannot these be class attributes?

Reply:
Done